app/models/vacation.rb

```diff
@@ -1,10 +1,15 @@
-class Vacation < ApplicationRecord
+class Vacation < ApplicationRecord  
+
+  MAX_SIMULTANEOUS_VACATIONS = 5
 
   belongs_to :employee
+  belongs_to :position
   
   validates :start_date, presence: true
   validates :end_date, presence: true
+  validates :position_id, presence: true
```

Comment:
I guess when we have `belongs_to :position`, we don't need `validates :position_id`